ingress/README.md

```diff
@@ -0,0 +1,103 @@
+# UpCloud Kubernetes Service with Nginx Ingress Controller
+
+UpCloud Kubernetes Service uses UpCloud Managed Load Balancer to provide Service Type: Load Balancer capabilities to Kubernetes. By default, the deployed Load Balancer will use Development Plan and it will be in HTTP Mode. For normal usage this is fine, but in conjuction with an Ingress Controller such as Nginx Ingress, you might need to change the Load Balancer settings. This example shows how you can use annotations to modify the Load Balancer that Nginx deploys during installation to support a production grade plan and switch the mode to TCP. The mode switch is needed if you want to terminate TLS in the Ingress Controller. The UpCloud Managed Load Balancer also supports the TLS to be terminated in the load balancer itself, in which case HTTP mode is sufficient.
```

Comment:
Not sure if it's even relevant, but isn't the service type "LoadBalancer" and not "Load Balancer"? No need to fix if you don't think this is all that valid. :)

Reply:
no you are right, fixed :)